submariner-operator/move-subctl-to-its-own-repo.md

```diff
@@ -0,0 +1,69 @@
+# Move subctl to its own repository
+
+## Summary
+
+Currently, `subctl` code lives in the `submariner-operator` repository. Both `subctl` and the Operator require different sets of
+dependencies which might conflict with improving either or both. For example, `subctl` can't be `go install`-ed currently because the
+Operator dependencies don't allow it. It also makes Operator upgrades difficult.
+
+## Proposal
+
+The proposal is to move `subctl` out of the `submariner-operator` repository to its own repository. Everything specific to `subctl` will be
+moved to the new `subctl` repository. The `subctl` project will depend on the Operator (_e.g._ for the CRDs, RBAC etc. which are defined in
+the Operator). The `submariner-operator` repository will only contain artifacts which the Operator framework generates and/or needs.
+
+### Pros
+
+* Operator framework upgrades would be simpler.
+* It's logically correct to separate the Operator and the library as they are 2 different deliverables and provide different ways to
+  install Submariner.
+* Testing criterias are different. We don't need `subctl`-related GHAs for Operator and vice versa.
```

Comment:
```suggestion
* Testing criteria are different. We don't need `subctl`-related GHAs for Operator and vice versa.
```
"Criteria" is already plural, "criterion" is the singular. I didn't know either, just learned this too FYI. :nerd_face: :+1: